GliaWidgets/CallVisualizer/ScreenSharing/Coordinator/ScreenSharingCoordinator.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+extension CallVisualizer {
+    final class ScreenSharingCoordinator: FlowCoordinator {
+        typealias ViewController = UIViewController
+        var delegate: ((DelegateEvent) -> Void)?
+
+        private let theme: Theme
+        private let environment: Environment
+        private var viewModel: ScreenSharingViewModel?
+        private var viewController: ScreenSharingViewController?
```

Comment:
I think it is fine to omit it reference to view controller here.

Reply:
a6dd3c3de3c9d0a64905b3c0de1f4f5242ad53c1